pkgs/development/ocaml-modules/getopt/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, fetchzip, stdenv, ocaml, findlib, ocamlbuild }:
+
+stdenv.mkDerivation {
+  pname = "getopt";
+  version = "20120615";
+
+  src = fetchzip {
+    url = https://download.ocamlcore.org/ocaml-getopt/ocaml-getopt/20120615/ocaml-getopt-20120615.tar.gz;
+    sha256 = "0bng2mmdixpmj23xn8krlnaq66k22iclwz46r8zjrsrq3wcn1xgn";
+  };
+
+  buildInputs = [
+    ocaml
+    findlib
+    ocamlbuild
+  ];
+
+  doCheck = true;
+  createFindlibDestdir = true;
+
+  meta = {
```

This is missing a `platforms` field.
```suggestion
  meta = {
    inherit (ocaml.meta) platforms;
```